File: src/main/java/com/anod/appwatcher/utils/Keyboard.java
package com.anod.appwatcher.utils;

import android.content.Context;
import android.view.View;
import android.view.inputmethod.InputMethodManager;

/**
 * @author algavris
 * @date 27/08/2016.
 */

public class Keyboard {

    public static void hide(View view, Context context)
    {
        // hide virtual keyboard
        InputMethodManager imm = (InputMethodManager) context.getSystemService(Context.INPUT_METHOD_SERVICE);
        imm.hideSoftInputFromWindow(view.getWindowToken(), 0);

    }
}

File: framework/src/main/kotlin/info/anodsplace/framework/view/Keyboard.kt
package info.anodsplace.framework.view

import android.content.Context
import android.view.View
import android.view.inputmethod.InputMethodManager

/**
 * @author Alex Gavrishev
 * *
 * @date 27/08/2016.
 */

object Keyboard {

    fun hide(view: View, context: Context) {
        // hide virtual keyboard
        val imm = context.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
        imm.hideSoftInputFromWindow(view.windowToken, 0)

    }
}
